pkgs/data/misc/media-types/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib, stdenv, fetchFromGitLab }:
+
+stdenv.mkDerivation rec {
+  pname = "media-types";
+  version = "8.0.0";
+
+  src = fetchFromGitLab {
+    domain = "salsa.debian.org";
+    owner = "debian";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-dl6XKxOF9YT9n/iSvZj1juTqSMaZuOF5i7Eb/VZZDFY";
+  };
+
+  installPhase = ''
+    install -D -m0444 mime.types $out/etc/mime.types
+    install -D -m0444 debian/bug-presubj $out/share/bug/media-types/presubj
+    install -D -m0444 debian/changelog $out/share/doc/media-types/changelog
+    install -D -m0444 debian/copyright $out/share/doc/media-types/copyright
+
+    # Add missing mime-types
+    sed -i $out/etc/mime.types \
+      -e '/^application\/x-wais-source\t\t\tsrc$/aapplication\/x-web-app-manifest+json\t\twebapp' \
+      -e '/^text\/vnd.net2phone.commcenter.command\t\tccc$/atext\/vnd.rim.location.xloc\t\t\txloc' \
+      -e 's/^text\/xml$/text\/xml\t\t\t\t\txsd rng/'
```

Comment:
Please do not hand pick extra media types. We also do not do that for mime.types.

Reply:
Eh, deleted.